dev/tests/unit/Magento/FunctionalTestFramework/DataGenerator/Handlers/PersistedObjectHandlerTest.php

```diff
@@ -416,10 +428,15 @@ public function testRetrieveEntityInValidField($name, $key, $value, $type, $scop
         $this->mockCurlHandler($jsonReponseOne);
         $handler->createEntity($stepKey, $scope, $name);
 
-        $this->expectException(\Magento\FunctionalTestingFramework\Exceptions\TestReferenceException::class);
-
         // Call method
         $handler->retrieveEntityField($stepKey, $invalidDataKey, $scope);
+
+        // validate log statement
+        TestLoggingUtil::getInstance()->validateMockLogStatement(
+            "warning",
```

Comment:
maybe change double quotes to single? Not the end of the world, but read that it avoids processing overhead for strings.

Reply:
Fixed